client/__tests__/e2e/e2e.test.ts

```diff
@@ -678,29 +677,33 @@ for (const option of options) {
       await assertGenesetExists(genesetName, page);
       await keyboardUndo(page);
       await assertGenesetDoesNotExist(genesetName, page);
+      // if we redo too quickly after undo, the shortcut handler capture the action
+      await page.waitForTimeout(500);
```

Comment:
ugh bp3

Reply:
I thought about including the timeout in the keyboardRedo helper, but didn't want to couple the two. Might create a `undoThenAssertThenRedoThenAssert` . Thoughts?

Reply:
Creating the util function SGTM! I think we can shorten the name of the function though. Maybe just `assertUndoRedo()` would be fine?